cli/scripts/monorepo.ts

```diff
@@ -184,6 +184,7 @@ importers:
           test: `${turboPath} run test`,
           lint: `${turboPath} run lint`,
           special: "echo root task",
+          args: 'node -e "console.log(process.argv)" --',
```

Comment:
Can you help me understand why we should include `--` here? I'm assuming that it's there for for a reason and I'd like to have it be a _bit_ more fleshed-out to help with detection of issues.

`node -e "console.log(process.argv)" --before-separator=thing -- --after-separator=thing`

We can also add additional scripts for "has separator" and "doesn't have separator."

Reply:
I had the trailing `--` so that we could detect when turbo was passing an unnecessary separator since Node swallows the first `--`:
```
olszewski@chriss-mbp cli % node -e "console.log(JSON.stringify(process.argv))" -- foo
["/Users/olszewski/.nvm/versions/node/v16.16.0/bin/node","foo"]
olszewski@chriss-mbp cli % node -e "console.log(JSON.stringify(process.argv))" -- -- foo
["/Users/olszewski/.nvm/versions/node/v16.16.0/bin/node","--","foo"]
```

Reply:
Wow, it's turtles all the way down. Can you add this information in as a comment?

I still would like to have a `before` and `after` sentinel argument here, just to monitor for behavior changes.

Reply:
Can you elaborate on having a `before`/`after` sentinel arguments for this script? 

Unsure if it relates but, once node encounters an arg it doesn't recognize it treats that and every arg after it as a pass through:
```
olszewski@chriss-mbp cli % node -e "console.log(JSON.stringify(process.argv))" before -- after             
["/Users/olszewski/.nvm/versions/node/v16.16.0/bin/node","before","--","after"]
olszewski@chriss-mbp cli % node -e "console.log(JSON.stringify(process.argv))" --zero-fill-buffers before         
["/Users/olszewski/.nvm/versions/node/v16.16.0/bin/node","before"]
olszewski@chriss-mbp cli % node -e "console.log(JSON.stringify(process.argv))"  before --zero-fill-buffers
["/Users/olszewski/.nvm/versions/node/v16.16.0/bin/node","before","--zero-fill-buffers"]
```

Reply:
Good callout that `node` itself blows up here. Thanks for that! This is fine as is.